README.md

```diff
@@ -1,95 +1,150 @@
-![Base](logo.webp)
+# Lisk
 
-# Base node
+[![License: Apache 2.0](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
+![GitHub repo size](https://img.shields.io/github/repo-size/liskhq/lisk-node)
+![GitHub issues](https://img.shields.io/github/issues-raw/liskhq/lisk-node)
+![GitHub closed issues](https://img.shields.io/github/issues-closed-raw/liskhq/lisk-node)
 
-Base is a secure, low-cost, developer-friendly Ethereum L2 built to bring the next billion users onchain. It's built on Optimism’s open-source [OP Stack](https://stack.optimism.io/).
+Lisk provides a cost-efficient, fast, and scalable Layer 2 (L2) network based on [Optimism (OP)](https://stack.optimism.io/). that is secured by Ethereum.
 
-This repository contains the relevant Docker builds to run your own node on the Base network.
+This repository contains information on how to run your own L2 node on the Lisk network.
 
-<!-- Badge row 1 - status -->
+## System requirements
 
-[![GitHub contributors](https://img.shields.io/github/contributors/base-org/node)](https://github.com/base-org/node/graphs/contributors)
-[![GitHub commit activity](https://img.shields.io/github/commit-activity/w/base-org/node)](https://github.com/base-org/node/graphs/contributors)
-[![GitHub Stars](https://img.shields.io/github/stars/base-org/node.svg)](https://github.com/base-org/node/stargazers)
-![GitHub repo size](https://img.shields.io/github/repo-size/base-org/node)
-[![GitHub](https://img.shields.io/github/license/base-org/node?color=blue)](https://github.com/base-org/node/blob/main/LICENSE)
+The following system requirements are recommended to run Lisk L2 node.
 
-<!-- Badge row 2 - links and profiles -->
+### Memory
 
-[![Website base.org](https://img.shields.io/website-up-down-green-red/https/base.org.svg)](https://base.org)
-[![Blog](https://img.shields.io/badge/blog-up-green)](https://base.mirror.xyz/)
-[![Docs](https://img.shields.io/badge/docs-up-green)](https://docs.base.org/)
-[![Discord](https://img.shields.io/discord/1067165013397213286?label=discord)](https://base.org/discord)
-[![Twitter Base](https://img.shields.io/twitter/follow/Base?style=social)](https://twitter.com/Base)
+- Modern multi-core CPU with good single-core performance
+- Machines with a minimum of 16 GB RAM (32 GB recommended).
 
-<!-- Badge row 3 - detailed status -->
+### Storage
 
-[![GitHub pull requests by-label](https://img.shields.io/github/issues-pr-raw/base-org/node)](https://github.com/base-org/node/pulls)
-[![GitHub Issues](https://img.shields.io/github/issues-raw/base-org/node.svg)](https://github.com/base-org/node/issues)
+- Machines with a high performance SSD drive with at least 4 TB free
```

Comment:
maybe this is a recommended requirement instead of hard requirement?

Reply:
In the official document of optimism, they also mention requirements regarding SSD.Ref: https://docs.optimism.io/builders/node-operators/tutorials/mainnet#ssd-capacity-requirements